api/background/utils.ts

```diff
@@ -10,7 +10,21 @@ import {
   WALLET_SETTINGS,
 } from "@/contexts/WalletManagerContext";
 
+const POPUP_WINDOW_WIDTH = 375;
```

Comment:
Can we export those for the `window.resizeTo()`?

Reply:
https://github.com/forbole/kastle/pull/54/commits/1f5f85c7f521fad5854796997c54c9c9175baf52